apis/v1alpha1/generator.yaml

```diff
@@ -72,6 +74,47 @@ resources:
         type: map[string]*string
     tags:
       ignore: true
+  InstanceProfile:
+    renames:
+      operations:
+        CreateInstanceProfile:
+          input_fields:
+            InstanceProfileName: Name
+        GetInstanceProfile:
+          input_fields:
+            InstanceProfileName: Name
+        DeleteInstanceProfile:
+          input_fields:
+            InstanceProfileName: Name
+    exceptions:
+      terminal_codes:
+        - InvalidInput
+    fields:
+      Path:
+        late_initialize: {}
```

Comment:
Since there is no way to update the path, let's mark it as immutable

Reply:
@a-hilaly Went ahead and made the change, but it seems like other IAM resources (e.g. Policy, Role, OIDCProvider) haven't specifically specified this as `is_immutable` even if those resources technically don't have code to update the path either. I wasn't quite sure if you wanted to keep things "unified" for now.